src/script/view_model/content/PreferencesAVViewModel.ts

```diff
@@ -196,7 +196,7 @@ export class PreferencesAVViewModel {
     await this.initiateDevices(mediaType);
   }
 
-  private async releaseDevices(mediaType: MediaType): Promise<void> {
+  public async releaseDevices(mediaType: MediaType): Promise<void> {
```

Comment:
`public` is the default, so no need to declare it.

Reply:
https://github.com/wireapp/wire-webapp/pull/9298/commits/8f08faae128071b7457ea81e5761fa7b5ef8336f